CSS/style.css

```diff
@@ -79,9 +81,214 @@ main {
 }
 
 .social-media-icons li {
-  padding: 0 20px;
+  padding: 1px 20px;
 }
 
 .scroll-down-icon {
-  margin: 64px 0 22px;
+  margin: 64px 1px 22px;
+}
+
+.works-section {
+  display: flex;
+  text-align: center;
+  flex-flow: column nowrap;
+  padding-left: 24px;
+  padding-right: 24px;
+}
+
+.works-section-title h2 {
+  max-width: 70%;
+  margin: 0 auto;
+  font-size: 40px;
+  color: var(--primary);
+  font-weight: var(--weight);
+  line-height: 52px;
+}
+
+.works-section-title h2::after {
+  content: " ";
+  display: block;
+  max-width: 48px;
+  height: 4px;
+  margin: 24px auto 62px;
+  background-color: var(--secondary);
+}
+
+.card-container {
+  display: grid;
+  width: auto;
+  grid-gap: 24px;
+  grid-template-columns: repeat(1, 1fr);
+  grid-auto-rows: minmax(474px, auto);
+}
+
+.card-container .card {
+  min-width: 327px;
+  height: 474px;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  justify-self: center;
+  display: inline-flex;
+  align-items: center;
+  flex-flow: column nowrap;
+}
+
+.card h3 {
+  padding: 16px;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  color: #3a4a42;
+}
+
+.card img {
+  background: var(--background);
+  height: 252px;
+  min-width: 100%;
+}
+
+.card ul,
+.cards ul {
+  width: 100%;
+  display: flex;
+  align-items: flex-start;
+  justify-content: center;
+  flex-wrap: wrap;
+  text-align: center;
+  list-style-type: none;
+}
+
+.card .program-lang li {
+  height: 32px;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  border-radius: 4px;
+  padding: 8px 12px;
+  margin: 0 4px;
+  color: #3a4a42;
+  background: var(--background);
+}
+
+.card button {
+  margin: 29px 0 24px;
+  height: 48px;
+  width: 122px;
+  padding: 12px;
+  font-size: 17px;
+  line-height: 24px;
+  font-weight: 500;
+  color: #fff;
+  border: none;
+  border-radius: 4px;
+  background-color: var(--secondary-btn);
+}
+
+.about-section {
+  padding-bottom: 59px;
+}
+
+.about-section .about {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  width: 100%;
+  background-image: url('../images/illustration-\ about\ me-\ mobile@2x.png');
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: center left;
+}
+
+.about h2 {
+  min-width: 100%;
+  margin: 0 auto;
+  padding: 68px 0 48px;
+  font-size: 40px;
+  line-height: 52px;
+  font-weight: var(--weight);
+  color: var(--primary);
+  background-image: url('../images/Mask\ Group@2x.png');
```

Comment:
- [ ] This is great to add the background image instead of just the image tags as background, but we shouldn't actually add that background to the h2 tag, instead,  we should set the background to a container.

Reply:
Thank you so much for your deep review.